rllib/agents/ddpg/ddpg_torch_policy.py

```diff
@@ -65,9 +66,18 @@ def ddpg_actor_critic_loss(policy, model, _, train_batch):
                 mean=torch.zeros(policy_tp1.size()),
                 std=policy.config["target_noise"]), -target_noise_clip,
             target_noise_clip)
-        policy_tp1_smoothed = torch.clamp(policy_tp1 + clipped_normal_sample,
-                                          policy.action_space.low.item(0),
-                                          policy.action_space.high.item(0))
+
+        policy_tp1_smoothed = torch.min(
+            torch.max(
+                policy_tp1 + clipped_normal_sample,
```

Comment:
You must also make sure that `clipped_normal_sample` is on the same device than `policy_tp1`, since currently `clipped_normal_sample` is always on 'cpu'.

Reply:
This was fixed.